stdlib.sh

```diff
@@ -166,6 +169,18 @@ source_env() {
   popd > /dev/null
 }
 
+# Usage: watch_file <filename>
+#
+# Adds <path> to the list of files that direnv will watch for changes - useful when the contents
+# of a file influence how variables are set - especially in direnvrc
+#
+watch_file() {
+  local file=${1/#\~/$HOME}
+
+  eval $($direnv watch $file)
```

Comment:
quoted `"$file"`.

Shell escaping is tricky. I always use shellcheck on all of my scripts because of that.